Sources/YCarousel/Enums/CarouselViewController+Strings.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  CarouselViewController+Strings.swift
+//  YCarousel
+//
+//  Created by Sahil Saini on 05/04/23.
+//  Copyright © 2023 Y Media Labs. All rights reserved.
+//
+
+import Foundation
+import YCoreUI
+
+extension CarouselViewController {
```

Comment:
This is missing a `CarouselViewController+StringsTests.swift` file that iterates through the enum and confirms that the string resources load.